packages/payment-processor/overrides/jest/index.d.ts

```
@@ -0,0 +1,1779 @@
+     */
+    stringContaining(str: string): any;
+
+    not: InverseAsymmetricMatchers;
+
+    setState(state: object): void;
+    getState(): MatcherState & Record<string, any>;
+  }
+
+  type JestMatchers<T> = JestMatchersShape<Matchers<void, T>, Matchers<Promise<void>, T>>;
+
+  type JestMatchersShape<TNonPromise extends {} = {}, TPromise extends {} = {}> = {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)